exporter/sumologicexporter/exporter.go

```diff
@@ -16,41 +16,45 @@ package sumologicexporter
 
 import (
 	"context"
-	"errors"
-	"fmt"
+	"net/http"
 
+	"github.com/pkg/errors"
```

Comment:
Please avoid using this pkg :)

Reply:
Can you explain why?

Quick search reveals numerous usages of this package: https://github.com/open-telemetry/opentelemetry-collector-contrib/search?q=github.com%2Fpkg%2Ferrors in other exporters and receivers.

Any other alternative is suggested to be used in otelcol repositories?

Reply:
As you can see even in this file `fmt.Errorf` has exactly the same API and is a language native method.

Reply:
Done